app/controllers/grades_controller.rb

```diff
@@ -88,6 +88,27 @@ def view_my_scores
     @summary = sum.summary
     @avg_scores_by_round = sum.avg_scores_by_round
     @avg_scores_by_criterion = sum.avg_scores_by_criterion
+
+
+
+
+    @review_mappings = SelfReviewResponseMap.where(reviewer_id: params[:id])
```

Comment:
The variable naming is not correct. @review_mappings variable is not a response map, but a SelfReviewResponseMap.